android/app-newm/src/main/java/io/newm/screens/library/NFTLibraryViewModel.kt

```diff
@@ -1,56 +1,48 @@
-@file:OptIn(ExperimentalCoroutinesApi::class)
-
 package io.newm.screens.library
 
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
-import io.newm.shared.models.Song
+import io.newm.shared.login.repository.KMMException
+import io.newm.shared.repositories.NFTTrack
 import io.newm.shared.usecases.WalletConnectUseCase
 import io.newm.shared.usecases.WalletNFTSongsUseCase
-import kotlinx.coroutines.ExperimentalCoroutinesApi
-import kotlinx.coroutines.flow.MutableStateFlow
-import kotlinx.coroutines.flow.SharingStarted
-import kotlinx.coroutines.flow.StateFlow
-import kotlinx.coroutines.flow.filterNotNull
-import kotlinx.coroutines.flow.flatMapLatest
-import kotlinx.coroutines.flow.mapLatest
-import kotlinx.coroutines.flow.stateIn
+import kotlinx.coroutines.flow.*
 import kotlinx.coroutines.launch
 
 class NFTLibraryViewModel(
-    private val useCase: WalletNFTSongsUseCase,
-    private val walletConnectManager: WalletConnectUseCase
+    private val walletConnectUseCase: WalletConnectUseCase,
```

Comment:
We shouldn't use the term "WalletConnect" since we might add the actual "WalletConnect" protocol at some point.  "ConnectWallet" would be less ambiguous.